src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/controllers/GenerateRepresentedClaimCopyTest.java

```diff
@@ -27,6 +33,9 @@ public void shouldReturnPdfDocumentIfEverythingIsFine() throws Exception {
         given(pdfServiceClient.generateFromHtml(any(), any()))
             .willReturn(PDF_BYTES);
 
+        given(documentUploadClientApi.upload(anyString(), any(List.class)))
+            .willReturn(DataHelper.documentManagementUploadResponse());
```

Comment:
Use a static import, it will read better.

Reply:
done